src/collectd.conf.pod

```diff
@@ -3322,6 +3322,28 @@ TCP-Port to connect to. Defaults to B<411>.
 
 =back
 
+=head2 Plugin C<mcelog>
+
+The C<mcelog plugin> uses mcelog to retrieve machine check exceptions, etc.
+
+By default the plugin connects to B<"/var/run/mcelog-client"> to check if the
+mcelog server is running. When the server is running, the plugin will tail the
+specified logfile to retrieve machine check exception information and send a
+notification with the details from the logfile. The plugin will use the mcelog
+client protocol to retrieve memory errors.
```

Comment:
At the moment mcelog's client/server protocol  returns only memory related MCE's, see http://www.mcelog.org/protocol.html. That's why I used 'memory errors' term. Would you like to extend it to  all MCE,  or maybe  use term 'memory related machine check exceptions' ?